flink-runtime-web/web-dashboard/package.json

```
@@ -6,7 +6,9 @@
     "start": "ng serve",
     "build": "ng build --configuration production --base-href ./",
     "test": "ng test",
-    "lint": "ng lint",
+    "lint": "eslint --cache src --ext .ts,.html & stylelint \"**/*.less\"",
+    "lint:fix": "eslint --fix --cache src --ext .ts,.html & stylelint \"**/*.less\" --fix",
```

You're actually starting eslint as a background task and then stylelint. This will cause eslint to finish after the process has already exited, and cause a zero return code even if eslint found issues.

```suggestion
    "lint": "eslint --cache src --ext .ts,.html && stylelint \"**/*.less\"",
    "lint:fix": "eslint --fix --cache src --ext .ts,.html && stylelint \"**/*.less\" --fix",
```

---

Yep, this was a potential problem. Thanks for pointing it out.